### PR TITLE
Proposal: Prune Translations and Add Official Translator Position to Governance

### PR DESCRIPTION
**Pull request by titusfortner**
#### Problem
Trying to reorganize and update the English documents becomes extremely unwieldy when it requires updating aliases, creating new files and marking translation pages potentially out of date for 8 additional languages.

#### Proposal 1
PLC/TLC updates governance document to create a new role for designated translators. I've suggested wording in this PR that will likely need to be adjusted.

@luisfcorreia has agreed to maintain pt-br translations
@alaahong has agreed to maintain the zh-cn translations
@kzhirata has agreed to maintain the ja translations

#### Proposal 2

We only maintain translations that have official maintainers.
@boris779 let us know if you want to keep maintaining German translations. 

At this time that means we archive translations for `nl`, `ko`, `de`, `fr`, and `es`. We can put them into separate branches so if someone volunteers to take on responsibility for one of them they have a starting point if they want.

Before we act on this, we should solicit feedback from the community, and we should have Proposal 1 agreed to before we do that.

#### Supporting Data

Here's the number of visitors on the landing page for a given translation in the past 2 weeks, and the percent of documentation pages translated:
| language | visitors | translated |
| --------- | ------- | ------- |
| ja | 1,700 | 86% |
| zh-cn | 1,500 | 86% |
| es| 409 |  42% |
| fr | 203 | 31% |
| pt-br | 189 | 87% |
| de | 88 | 65% |
| ko | 79 | 68% |
| nl | 22 | 6% |

### CODE REVIEW

**Comment by kzhirata** [comment]
@titusfortner Thank you for your translation notice. I will be responsible for translating the following PR little by little.
https://github.com/SeleniumHQ/seleniumhq.github.io/pull/843 
And I just joined the slack channel #selenium-docs .

**Comment by titusfortner** [comment]
@kzhirata that PR is mostly just reorganization, the main content updates will start coming after that. :)

**Comment by titusfortner** [comment]
Also of potential interest here, I've added a language "Other" for the landing page that contains:

> Is there another translation you'd like to see? We're only supporting translations for which we have 
> a dedicated translator. If you'd like to volunteer to be a translator, read how you can help

**Comment by alaahong** [comment]
Hi @titusfortner 
It's my honor to take this responsibility,  just joint the slack channel #selenium-docs , other place else need please  let me know. 
Thanks.

File: website_and_docs/content/project/governance/_index.html [hunk 1]
```diff
@@ -111,6 +111,28 @@ <h4 class="pt-4">Process for becoming a triager</h4>
...
 
+    <div class="pb-5">
+      <h3>Translators</h3>
+
+      To better support our international community, we provide translations of our documentation. A translator is
+      responsible for all content in a given language. This includes creating and updating content, helping ensure
+      that content created in one language is flagged for distributions to other languages, and managing any
+      Pull Requests relevant to those language files.
+
+      <h4 class="pt-4">Process for becoming a triager</h4>
```
Comment (diemol) [inline website_and_docs/content/project/governance/_index.html]:
```suggestion
      <h4 class="pt-4">Process for becoming a translator</h4>
```

**Review by diemol — CHANGES_REQUESTED** [review]
Aside from the suggestion, 👍 

Thanks, @titusfortner!

**Review by diemol — APPROVED** [review]

**Pull request merged**